manim/mobject/opengl_compatibility.py

```diff
@@ -15,7 +15,9 @@ class ConvertToOpenGL(ABCMeta):
     spurious errors.
     """
 
-    def __new__(cls, name, bases, namespace):
+    _converted_classes = []
+
+    def __new__(mcls, name, bases, namespace):
```

Comment:
Sure: it is to make clearer that the first argument passed to `__new__` is not the class that is being constructed (because, well, the class object doesn't exist at that point) -- but rather the metaclass.